src/inject/index.ts

```diff
@@ -224,4 +225,69 @@ if (__TEST__) {
             id: null,
         }));
     };
+
+    if (__FIREFOX__) {
+        function expectPageStyles(data: any) {
+            const errors = [];
+            const expectations = Array.isArray(data[0]) ? data : [data];
+            for (const [selector, cssAttributeName, expectedValue] of expectations) {
+                let element: Element = document as unknown as Element;
+                const selector_ = Array.isArray(selector) ? selector : [selector];
+                for (const part of selector_) {
+                    if (element instanceof HTMLIFrameElement) {
+                        element = (element as any).contentDocument;
+                    }
+                    if (part === 'document') {
+                        element = (element as any).documentElement;
+                    } else {
+                        element = element.querySelector(part);
+                    }
+                }
+                const style = getComputedStyle(element);
+                const realValue = style[cssAttributeName];
+                if (realValue !== expectedValue) {
+                    errors.push({
+                        selector,
+                        cssAttributeName,
+                        expectedValue,
+                        realValue,
+                    });
+                }
+            }
+            return errors;
+        }
+
+        socket.onmessage = (e) => {
+            function respond(data: any) {
+                socket.send(JSON.stringify({id, data}));
+            }
+
+            const {id, data, type} = JSON.parse(e.data);
+            switch (type) {
+                case 'firefox-eval': {
+                    const result = eval(data);
```

Comment:
This should be probably wrapped into
```typescript
if (__DEBUG__) {
    const result = eval(data);
}
```
or
```typescript
if (__TEST__) {
    const result = eval(data);
}
```

I'm afraid it will not pass the review if included into build.